x-pack/plugins/timelines/public/hooks/use_add_to_case.ts

```diff
@@ -239,3 +241,17 @@ export const useAddToCase = ({
     isCreateCaseFlyoutOpen,
   };
 };
+
+export function normalizedEventFields(event?: TimelineItem) {
+  const ruleId = event && event.data.find(({ field }) => field === ALERT_RULE_ID);
+  const ruleUuid = event && event.data.find(({ field }) => field === ALERT_RULE_UUID);
+  const ruleName = event && event.data.find(({ field }) => field === ALERT_RULE_NAME);
+  const ruleIdValue = ruleId && ruleId.value && ruleId.value[0];
```

Comment:
is it possible value could have a length of 0? perhaps consider :
```suggestion
  const ruleIdValue = getOr('', 'value[0]', ruleId);
```